templates/etc/fail2ban/jail.local.j2

```diff
@@ -105,7 +105,7 @@ action = {{ fail2ban_action }}
 {% for service in fail2ban_services %}
 [{{ service.name }}]
 enabled = {{ service.enabled | default(true) | bool | to_json }}
-{% for option, value in service.items() %}
+{% for option, value in service.items()|sort %}
```

Comment:
Shouldn't this be [dictsort](http://jinja.pocoo.org/docs/2.10/templates/#dictsort)?

Reply:
it seems that using dictsort give me the same result.
Using this : `{% for option, value in service|dictsort %}`

I am still new in the jinja2 templating world and I don't know which solution is the best suited here. But I think that, as you propose, dictsort seems to be made for this kind of situation, dealing with dict.

I will let you choose the right way here ;-)

Reply:
I'll have a look at it

Reply:
https://www.tutorialspoint.com/python/dictionary_items.htm

> `items()` returns a list of dict's (key, value) `tuple` pairs

So `sort` seems to be correct